iceoryx_posh/testing/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+
+# Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Now this becomes weird. I am pretty sure that `iox_add_library` was introduced this year and there is nothing else left in here so why keep the bosch copyright?

But you can keep it if you want ... I find it just more and more silly that we have to drag this everywhere.